packages/forge2d/lib/src/dynamics/body_def.dart

```diff
@@ -87,6 +88,6 @@ class BodyDef {
   /// Does this body start out active?
   bool active;
 
-  /// Experimental: scales the inertia tensor.
```

Comment:
@spydon do we know why this was mark as experimental?

Reply:
No idea, that comment was there way before I started working on this. Maybe check the git log for that line of there are any hints im there?